src/main/groovy/edu/oregonstate/mist/catalogapitest/db/CourseDAO.groovy

```diff
@@ -35,38 +56,70 @@ public interface CourseDAO extends Closeable {
             """)
     Course getByCrn(@Bind("crn") Integer crn)
 
-    // Get specific instance number
-    // FIXME: possible race condition
+    /**
+     * GETs specific course object instance number using Oracle sequences.
+     *
+     * @return Updated sequence number
+     */
     @SqlQuery("""
             SELECT COURSES_SEQ.CURRVAL FROM DUAL
             """)
     Integer getLatestCidNumber()
 
-    // PUT
+    /**
+     * PUTs a course object.
+     *
+     * @param crn
+     * @param courseName
+     * @param instructor
+     * @param day
+     * @param time
+     * @param location
+     */
     @SqlUpdate( """
               UPDATE COURSES
               SET COURSENAME = :courseName, INSTRUCTOR = :instructor, DAY = :day, TIME = :time, LOCATION = :location
               WHERE CRN = :crn
               """)
-    void putByCrn(@Bind("crn") Integer crn, @Bind("courseName") String courseName, @Bind("instructor") String instructor, @Bind("day") String day,
-                  @Bind("time") String time, @Bind("location") String location)
-    
-    // POST
-    @SqlUpdate("""INSERT INTO COURSES (CRN, COURSENAME, INSTRUCTOR, DAY, TIME, LOCATION)
-                  values (:crn, :courseName, :instructor, :day, :time, :location)""")
+    void putByCrn(@Bind("crn") Integer crn, @Bind("courseName") String courseName, @Bind("instructor") String instructor,
+                  @Bind("day") String day,  @Bind("time") String time, @Bind("location") String location)
```

Comment:
this and similar methods would be more readable if each argument were on its own line